packages/create-project/commands/init/index.js

```diff
@@ -112,61 +113,90 @@ module.exports = class Project extends Generator {
+    const packageManager = isYarn ? 'yarn' : 'npm';
+    const install = isYarn ? 'add' : 'install';
+    const devFlag = isYarn ? '--dev' : '--save-dev';
     const { dependencies, devDependencies } = this._getDependencies();
 
     this.log(`\n${chalk.green('success')} Saved package.json`);
-    process.chdir(this.options.directory);
 
     if (dependencies) {
       this.log(`\n${chalk.green('Installing dependencies:')} ${chalk.yellow(dependencies.join(', '))}`);
-      this.spawnCommandSync(installer, [argument, ...dependencies], { stdio: 'inherit' });
+      this.spawnCommandSync(packageManager, [install, ...dependencies], {
+        cwd: this.options.directory,
+        stdio: this.options.stdio,
+        env: process.env
+      });
     }
 
     if (devDependencies) {
-      this.log(`\n${chalk.green('Installing dev-dependencies:')} ${chalk.yellow(devDependencies.join(', '))}`);
-      this.spawnCommandSync(installer, [argument, development, ...devDependencies], { stdio: 'inherit' });
+      if (process.env.NODE_ENV === 'test') {
+        const remote = devDependencies.filter(d => !d.includes('@neutrinojs'));
+        const local = devDependencies.filter(d => d.includes('@neutrinojs'));
+
+        if (remote.length) {
+          this.log(`\n${chalk.green('Installing remote devDependencies:')} ${chalk.yellow(remote.join(', '))}`);
+          this.spawnCommandSync(packageManager, [install, devFlag, ...remote], {
+            stdio: this.options.stdio,
+            env: process.env,
+            cwd: this.options.directory
+          });
+        }
+
+        if (local.length) {
+          this.log(`\n${chalk.green('Linking local devDependencies:')} ${chalk.yellow(local.join(', '))}`);
+          this.spawnCommandSync('yarn', ['link', ...local], {
+            stdio: this.options.stdio,
+            env: process.env,
+            cwd: this.options.directory
+          });
+        }
+      } else {
+        this.log(`\n${chalk.green('Installing devDependencies:')} ${chalk.yellow(devDependencies.join(', '))}`);
+        this.spawnCommandSync(packageManager, [install, devFlag, ...devDependencies], {
+          stdio: this.options.stdio,
+          env: process.env,
+          cwd: this.options.directory
+        });
+      }
     }
 
     if (this.data.linter) {
-      this.spawnCommandSync('neutrino', ['lint', '--fix'], { stdio: 'ignore' });
+      this.spawnCommandSync('neutrino', ['lint', '--fix'], { stdio: this.options.stdio, cwd: this.options.directory });
     }
   }
 
   end() {
-    this.log(`\n${chalk.green('Success!')} Created ${chalk.cyan(this.options.directory)} at ${chalk.yellow(process.cwd())}`);
-    this.log(`To get started, change your current working directory to: ${chalk.cyan(this.options.directory)}`);
+    this.log(`\n${chalk.green('Hooray, I successfully created your project!')}`);
+    this.log(`To get started, change your current working directory to:\n  ${chalk.cyan(this.options.directory)}`);
```

Comment:
"Hooray" :) 

If the user is installing an app, what do you think if we log something like:

"To start your application, run `yarn start`" (or `npm start` if `isYarn` evaluates to false)

Reply:
I like it.